WMFData/Sources/WMFData/Data Controllers/Article Tabs/ArticleTabsDataController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ArticleTabsDataController.swift
+//  WMFData
+//
+//  Created by Grey Olson on 4/30/25.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+@objc public class ArticleTabsDataController: NSObject {
+    
+    public let coreDataStore: WMFCoreDataStore
+    private let userDefaultsStore: WMFKeyValueStore?
+    private let developerSettingsDataController: WMFDeveloperSettingsDataControlling
+    
+    public init(coreDataStore: WMFCoreDataStore? = WMFDataEnvironment.current.coreDataStore, userDefaultsStore: WMFKeyValueStore? = WMFDataEnvironment.current.userDefaultsStore, developerSettingsDataController: WMFDeveloperSettingsDataControlling = WMFDeveloperSettingsDataController.shared) throws {
+
+        guard let coreDataStore else {
+            throw WMFDataControllerError.coreDataStoreUnavailable
+        }
+        self.coreDataStore = coreDataStore
+        self.userDefaultsStore = userDefaultsStore
+        self.developerSettingsDataController = developerSettingsDataController
+    }
+    
+    // MARK: Entry point
+
+    @objc public var shouldShowArticleTabs: Bool {
+        get {
+            return (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.developerSettingsArticleTab.rawValue)) ?? true
```

Comment:
Try defaulting to false here instead of true - I saw the tabs icon when I first launched I think because of this.